docs/CONTRIBUTING.md

```diff
@@ -1,6 +1,52 @@
-As an open source project we welcome contributions of many forms. Please reach out before starting
-work on any major code changes. This will ensure we avoid duplicating work, or that
-your code can't be merged due to a rapidly changing code base. If you would like support
-for a library that is not listed, [contact support][1] to share a request.
+---
 
-[1]: https://docs.datadoghq.com/help
+<p align="center">
+  <strong>
+    <a href="https://github.com/open-telemetry/community#net-auto-instrumentation">Getting Involved<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://gitter.im/open-telemetry/opentelemetry-dotnet-auto-instr">Getting In Touch<a/>
+  </strong>
+</p>
+
+---
+
+# Contributing Guide
+
+We'd love your help!
+
+## Report a bug or requesting feature
+
+Reporting bugs is an important contribution. Please make sure to include:
+
+* Expected and actual behavior
+* OpenTelemetry, OS, and .NET versions you are using
+* If possible, steps to reproduce
+
+## How to contribute
```

Comment:
every other repo talks about creating forks, creating branch and submit PR. 
We need to specify it here as well, right ?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/master/CONTRIBUTING.md#how-to-send-pull-requests This part I mean.

Reply:
Right, I just didn't want to do it right now because of the conversation going on at https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/5